README.md

```diff
@@ -7,70 +7,33 @@ debt in the JS ecosystem.
 Our goal is to provide a cleaner, faster and simpler dependency tree in the JS
 ecosystem for those who are concerned with it.
 
-## The problem
+## How to help
```

Comment:
This section kind of duplicates the next one. The only one here that isn't below is "needs alternative". I wonder if we can remove this section and potentially add that one below